packages/react/src/index.js

```diff
@@ -72,7 +74,15 @@ export ListItem from './components/ListItem';
 export Loading from './components/Loading';
 export Modal from './components/Modal';
 export ModalWrapper from './components/ModalWrapper';
-export MultiSelect from './components/MultiSelect';
+import MultiSelect from './components/MultiSelect';
+if (__DEV__) {
```

Comment:
Would it make sense to put these directly on the component export itself instead of in the entrypoint file? This could help out with the direct import use-case, potentially, and catch any usage where we might be using these components from a relative path.

Reply:
@joshblack yes - I just pushed an update moving this to the exports themselves instead of the entrypoint file. Let me know what you think, I had to modify the syntax of the exports a bit.